### PR TITLE
    Reintroduce kolla into startup path for neutron-api

### PR DESCRIPTION
**Pull request by booxter**
This is to keep neutron-api in line with [1] until we decide otherwise.

Keeping kolla has several drawbacks, specifically:

1. allowPrivilegeEscalation has to be true for sudo.
2. dropCapabilities cannot be ALL because kolla_set_configs failed to change to root gid.

This patch may be revisited later if we decide to tighten privileges further by patching our kolla fork, or removing kolla completely from the startup path. For now, we'll have to stick to common approach.

The discussion over kolla place can be continued in jira [2].

[1] https://github.com/openstack-k8s-operators/docs/blob/main/service_config.md
[2] https://issues.redhat.com/browse/OSPRH-2240

### CODE REVIEW

File: api/bases/neutron.openstack.org_neutronapis.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
```
Comment (slawqo) [inline api/bases/neutron.openstack.org_neutronapis.yaml:6]: why this change?

File: config/crd/bases/neutron.openstack.org_neutronapis.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
```
Comment (slawqo) [inline config/crd/bases/neutron.openstack.org_neutronapis.yaml:6]: and same here

**Comment by karelyatin** [comment]
lgtm, the failures are related to patch so needs to be fixed

**Review by karelyatin — COMMENTED** [review]
/lgtm
